src/app/main/component/events/components/event-editor/event-editor.component.spec.ts

```
@@ -0,0 +1,85 @@
+      ],
+      schemas: [NO_ERRORS_SCHEMA]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(EventEditorComponent);
+    component = fixture.componentInstance;
+    fb = TestBed.inject(FormBuilder);
+
+    fixture.detectChanges();
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)